tests/integ/scala/test_dataframe_copy_into.py

```diff
@@ -197,6 +197,19 @@ def test_copy_csv_basic(session, tmp_stage_name1, tmp_table_name):
         sort=False,
     )
 
+    # test statement params
+    df.copy_into_table(tmp_table_name, _statement_params={"KEY": "VALUE"})
```

Comment:
hey team, do you know if there're some existing statement level parameters I could test that would change the query behavior, want to make sure this _statement_params is passed to the service side

Reply:
How about using the partner information that Bradley discussed with us?

Reply:
I used `DATE_INPUT_FORMAT` + `SF_PARTNER` in df tests to verify that statement level params is sent to the service.